src/SnapKeyring.ts

```diff
@@ -912,6 +912,7 @@ export class SnapKeyring extends EventEmitter {
         address: account.address.toLowerCase(),
         metadata: {
           name: '',
+          importTime: 0,
```

Comment:
As discussed together, it's ok because:
> - `importTime` is more the responsibility of `AccountsController`
> - 0 acts as a falsy value, so the `AccountsController` would know to update it

cc: @montelaidev